crates/sui-graphql-rpc/src/types/owner.rs

```diff
@@ -437,9 +449,15 @@ impl OwnerImpl {
         name: DynamicFieldName,
     ) -> Result<Option<DynamicField>> {
         use DynamicFieldType as T;
-        DynamicField::query(ctx.data_unchecked(), self.address, name, T::DynamicField)
-            .await
-            .extend()
+        DynamicField::query(
```

Comment:
Dynamic Field and Dynamic Object Field point queries also need to be bounded by their parent version.